### DISCUSSION TITLE
Drag and drop (dnd) inside modal

### DISCUSSION THREAD

**Discussion by yamatano18** [mantinedev/mantine#2892] [Q&A]
Hello,
Has anyone tried to create drag and drop (dnd) list inside a modal? The position of the dragged element is not determined correctly and there is an offset of the dragged element. When the modal size is height: 100vw and width: 100vh, everything works correctly.

**Comment by delateurj** [comment]
Couple of years later , but seeing same thing when using Chessboard.js and its drag and drop functionality.  Works okay if I use the full screen option of Modal.  If someone happens to know if way to make work without going fullscreen would appreciate it.  Loving the library, only issue so far and not a big one,.

**Reply by telmen** [reply]
@delateurj Hi, have you resolved it?
